pages/settings/profile.tsx

```diff
@@ -18,12 +19,23 @@ export default function Settings(props) {
   const descriptionRef = useRef<HTMLTextAreaElement>();
   const avatarRef = useRef<HTMLInputElement>();
   const hideBrandingRef = useRef<HTMLInputElement>();
+  const [selectedTheme, setSelectedTheme] = useState({ value: "" });
   const [selectedTimeZone, setSelectedTimeZone] = useState({ value: props.user.timeZone });
-  const [selectedWeekStartDay, setSelectedWeekStartDay] = useState(props.user.weekStart || "Sunday");
+  const [selectedWeekStartDay, setSelectedWeekStartDay] = useState({ value: "" });
 
   const [hasErrors, setHasErrors] = useState(false);
   const [errorMessage, setErrorMessage] = useState("");
 
+  const themeOptions = [
+    {value: 'light', label: 'Light'},
```

Comment:
Codacy has a fix for the issue: [Replace `value:·'light',·label:·'Light'` with `·value:·"light",·label:·"Light"·`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7734611)
```suggestion
    { value: "light", label: "Light" },
```